src/dataflow/utilities/aws.ts

```diff
@@ -50,3 +50,49 @@ export const getSignedUrl = (
     const requestUrl = protocol + "://" + host + uri + "?" + canonicalQuerystring;
     return requestUrl;
 };
+
+export function uploadProgram(programData: any): string {
+  let program = programData;
+  const endTimePad = 24 * 60 * 60 * 1000;
+  const newTimestamp = Date.now() + endTimePad;
+
+  if (!programData) {
```

Comment:
@dstrawberrygirl, is it useful to leave this in while testing or should we bail of there is no `programData`?

Reply:
Cool.

Reply:
Removed.